### PR TITLE
MSG Level1.5 NetCDF Reader (code and yaml file) for VIS/IR Channels

### PR DESCRIPTION
**Pull request by ColinDuff**
This PR contains the MSG Level1.5 Seviri NetCDF reader (code and yaml file), VIS/IR Channels only for now

I have tested the reader using full earth data , which was also resampled, and also with an ROI product.

All displayed correctly

Please check i have used dask correctly - not too up to speed with this.

### CODE REVIEW

**Comment by ColinDuff** [comment]
Further changes have been made to correct the Start/Time format and to fix the ROI area defintion calcualtion. Workaround was added to resolve xarry open_dataset not finding some incorrectly defined diemnsion variables.

File: satpy/readers/nc_seviri_l1b.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""SEVIRI netcdf format reader. """
+
+from satpy.readers.file_handlers import BaseFileHandler
+from satpy.readers.msg_base import (SEVIRICalibrationHandler,
```
[the remark this answers is not in the document]
Comment (djhoese) [reply satpy/readers/nc_seviri_l1b.py]: @ColinDuff what is the status of this PR? Looks like this import is still an issue?

[the remark this answers is not in the document]
Comment (ColinDuff) [reply satpy/readers/nc_seviri_l1b.py]: hi, satnum is now being used - dont know why its still giving this message

[the remark this answers is not in the document]
Comment (djhoese) [reply satpy/readers/nc_seviri_l1b.py]: Probably a buggy stickler run, no problem.

**Pull request merged**